src/main/java/ejedev/raidshamer/RaidShamerConfig.java

```diff
@@ -10,43 +10,87 @@ public interface RaidShamerConfig extends Config
 {
 
     @ConfigItem(
-        keyName = "captureOwnDeaths",
-        name = "Own Deaths Captured",
-        description = "Allows you to toggle on/off own death shaming",
-        position = 1
+            keyName = "captureOwnDeaths",
```

Comment:
Keep tabs to 4 spaces, same for all config items

Reply:
fixed in 5dfe6b6